exercises/go-counting/canonical-data.json

```diff
@@ -0,0 +1,172 @@
+{
+  "exercise": "go-counting",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Test for black territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+                    "  B  ",
+                    " B B ",
+                    "B W B",
+                    " W W ",
+                    "  W  "
+                 ],
+        "x": 0,
+        "y": 1
+      },
+      "expected": {
+        "stone": "BLACK",
+        "territory": [[0, 0], [0, 1], [1, 0]]
```

Comment:
Here is a suggestion I have. What if these were to be {"x": whatever, "y": whatever} instead?

For queen-attack that was suggested in #721 and implemented in #935
For robot-simulator that was suggested in #722 and implemented in #936

Does the same reasoning apply here? Why or why not?

Reply:
Well I use object notation for the input where there is just one x coordinate and one y coordinate. However, there can be many x and ys in the expected output, so array notation (which was also suggested in #721 and #722 seems cleaner, shorter, and simpler.

Reply:
If so, then let's define in a `"comments": ["The coordinates are ........."]` at the top level of the file which coordinate is which, otherwise it is not clear.

(Feel free to look at https://github.com/exercism/problem-specifications/blob/master/exercises/anagram/canonical-data.json#L4-L9 for example of syntax for the comment)

For me, my opinion will be that the order should be `[y, x]` because that matches the order in which one would index into the board (y selects which string in the array, x selects which character in the string)